rio_tiler/reader.py

```diff
@@ -50,37 +51,47 @@ def _read(
 
     Attributes
     ----------
-        src_dst : rasterio.io.DatasetReader
-            rasterio.io.DatasetReader object
-        height : int
-            Output height of the array.
-        width : int
-            Output width of the array.
-        indexes : list of ints or a single int, optional
-            Band indexes
-        out_window: rasterio.windows.Window, optional
-            Output window to read.
-        nodata: int or float, optional
-        resampling_method : str, optional
-            Resampling algorithm. Default is "bilinear".
-        force_binary_mask, bool, optional
-            If True, rio-tiler makes sure mask has only 0 or 255 values.
-            Default is set to True.
-        unscale, bool, optional
-            If True, apply scale and offset to the data array.
-            Default is set to False.
-        vrt_options: dict, optional
-            These will be passed to the rasterio.warp.WarpedVRT class.
+    src_dst: rasterio.io.DatasetReader
+        rasterio.io.DatasetReader object
+    height: int, optional
+        Output height of the array.
+    width: int, optional
+        Output width of the array.
+    indexes: list of ints or a single int, optional
+        Band indexes
+    out_window: rasterio.windows.Window, optional DEPRECATED
+        Output window to read.
```

Comment:
I think you should take out deprecations before the final 2.0 release

Reply:
yes that's the plan. If we done a Beta release, all the depreciation warning will go